doc/sphinx/source/releasenotes.rst

```diff
@@ -47,7 +49,7 @@ Interface changes
 New features
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * New HIP backend: ``/gpu/hip/ref``.
-* CeedQFunction support for user ``CUfunction``s in some backends
+* CeedQFunction support for user ``CUfunction``\s in some backends
```

Comment:
If we don't escape this s, the inline code display fails and we are left with ``